### PR TITLE
Fix mm's dependency on mistune

### PR DESCRIPTION
**Pull request by shubham1172**

### CODE REVIEW

File: tests/test_link_validation.py
Comment (paulyuk) [inline tests/test_link_validation.py:1]: this is the key to get builds going I believe.  We'll want to come back normal hours for a complete fix.

**Review by paulyuk — APPROVED** [review]

**Review by artursouza — CHANGES_REQUESTED** [review]
Let's avoid removing the tests.

**Comment by shubham1172** [comment]
> Let's avoid removing the tests.

Needs further investigation on what is causing the FakeHttpServer to hang in tests. It works fine locally, but gets stuck in CI.
The tests are limited to link validations which is fairly easy to test manually for now. However it's not a great practice to remove tests, so I am ok blocking this until we find a fix.


To unblock quickstarts, we can then pin to mm 0.6.0 and mistune 2.0.1.

**Comment by artursouza** [comment]
I found the issue. Let's see this run.

**Review by artursouza — APPROVED** [review]

**Pull request merged**